app/services/spree_shopify_importer/data_parsers/stock_locations/base_data.rb

```diff
@@ -0,0 +1,39 @@
+module SpreeShopifyImporter
+  module DataParsers
+    module StockLocations
+      class BaseData
+        def initialize(shopify_location)
+          @shopify_location = shopify_location
+        end
+
+        def attributes
+          @attributes ||= {
+            name: stock_location_name,
+            address1: @shopify_location.address1,
+            address2: @shopify_location.address2,
+            city: @shopify_location.city,
+            zipcode: @shopify_location.zip,
+            phone: @shopify_location.phone,
+            country: country,
+            state: state,
+            active: @shopify_location.active
+          }
+        end
+
+        private
+
+        def stock_location_name
+          "#{@shopify_location.name}/#{@shopify_location.id}"
+        end
+
+        def country
+          @country = Spree::Country.find_by(iso: @shopify_location.country_code, name: @shopify_location.country_name)
```

Comment:
why do you need to use `@country` here?

Reply:
I use it in state method and in attributes

Reply:
`@country ||= Spree::Country.find_by(iso: @shopify_location.country_code, name: @shopify_location.country_name)`

Reply:
discussed